cobalt/devinfra/kokoro/bin/cleanup.sh

```diff
@@ -7,4 +7,6 @@ set -x
 set +e
 git config --global --add safe.directory "${WORKSPACE_COBALT}"
 time git -C "${WORKSPACE_COBALT}" clean -dfx
```

Comment:
Instead of adding the lines below
how about making this `clean -dffx`?

Reply:
A commit/PR title for that could be
“Remove nested repositories after build on Kokoro”